### DISCUSSION TITLE
page load error when popup

### DISCUSSION THREAD

**Discussion by yw0lk1nni9ht** [cefsharp/CefSharp#4969] [General]
### Is there an existing issue for this?

- [X] I have searched both open/closed issues, no issue already exists.

### CefSharp Version

109.1.110

### Operating System

Windows 10

### Architecture

x64

### .Net Version

.net framework 4.7.2

### Implementation

WinForms

### Reproduction Steps

``` /*test.html*/
<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Open New Tab Example</title>
    <script>
        function openNewTab() {
            var newTab = window.open('https://www.etax.chinatax.gov.cn/', 'https://www.etax.chinatax.gov.cn/');
            // window.open('https://www.etax.chinatax.gov.cn/','_self');
        }
    </script>
</head>
<body>
    <button onclick="openNewTab()">点击我打开新标签页</button>
</body>
</html>
```

Use CefSharp.WinForms.Example，open test.html (code show above )，click the button. you can see the page is nothing.
but if you clear the cookie, and then change test.html code, use ```window.open('https://www.etax.chinatax.gov.cn/','_self');``` instead of ```var newTab = window.open('https://www.etax.chinatax.gov.cn/', 'https://www.etax.chinatax.gov.cn/');```, 
the page is load normal.
### Expected behavior

load website.
![error](https://github.com/user-attachments/assets/df887040-74e1-47b8-b662-f6375d5311a1)

### Actual behavior

request return http code 400.
![click](https://github.com/user-attachments/assets/dcc4349c-6caf-4f67-a8a8-a31b620e83d2)
![error11](https://github.com/user-attachments/assets/3b668422-4f60-42bf-87ad-cf1fbe932c17)

### Does this problem also occur in the CEF Sample Application

Yes using CefSharp.Winforms.Example. But it is normal in cefclient.exe

**Comment by amaitland** [comment]
> ### CefSharp Version
> 109.1.110

Version is only and unsupported, please don't open new issues unless you can reproduce the problem in the current supported version (M130 at time of writing)

Testing with https://www.w3schools.com/jsref/tryit.asp?filename=tryjsref_win_open and everything works fine in M130 using the `CefSharp.WinForms.Example`.

The `file:///` scheme has many security restrictions, should be avoided for anything other then displaying static text in my opinion.

**Comment by yw0lk1nni9ht** [comment]
Thank you very much for answering my questions. Following your advice, I tried reproducing the issue again using CefSharp.WinForms.Example, but without success. Eventually, I discovered the original problem: an extra space in "disable-site-isolation-trials" prevented the configuration from working correctly. 
Thank you once again for your kind assistance and support; I really appreciate it.
